docs/readthedocs/usersguide.md

```diff
@@ -1,224 +1,213 @@
-************
-User's Guide
-************
+# User's Guide
 
 
-Getting Started 
-===============
+## Getting Started
 
-The user's guide is here to give you a basic rundown of all the things you need to know to get you up and running. We will give you some information on how to install and setup SIMONA as well as some insights into configuring a simulation run. 
 
-Requirements
-============
+The user's guide is here to give you a basic rundown of all the things you need to know to get you up and running. We will give you some information on how to install and setup SIMONA as well as some insights into configuring a simulation run.
 
-To run and customize the project you need a Java Development Kit (JDK) installation. You can download it `here <https://adoptopenjdk.net/>`_ .
+## Requirements
 
+To run and customize the project you need a Java Development Kit (JDK) installation. You can download it [here](https://adoptopenjdk.net/).
 
-Installation
-============
 
-You can find and download the source code of the latest stable SIMONA version `here <https://github.com/ie3-institute/simona>`_. Go ahead and clone the repository using git:
+## Installation
 
-.. code-block:: none
 
-   $ git clone https://github.com/ie3-institute/simona.git
+You can find and download the source code of the latest stable SIMONA version [here](https://github.com/ie3-institute/simona). Go ahead and clone the repository using git:
 
+    $ git clone https://github.com/ie3-institute/simona.git
 
-Running a Standalone Simulation
-===============================
-   
-Running a standalone simulation is a matter of invoking the ``RunSimonaStandalone`` object. You can find it at ``./src/main/scala/edu.ie3/simona/main``. 
```

Comment:
This path should be `/src/main/scala/edu/ie3/simona/main`, would you mind changing it?